### PR TITLE
6.15.z-fix subscription search box xpath

### PR DESCRIPTION
**Pull request by vijaysawant**
**Problem**
Content - Subscription page search box xpath unable to locate due to wrong xpath

**Solution**
This updated xpath will work as expected.

**Note**
Robottelo changes from the PR [[Link]](https://github.com/SatelliteQE/robottelo/pull/15237) are not dependant these changes as I have updated some part of test case ( `test_positive_access_with_non_admin_user_with_manifest`) which won't search for any Subscription

### CODE REVIEW

**Review by sambible — APPROVED** [review]
@omkarkhatavkar can you review?

File: airgun/entities/subscription.py [hunk 1]
```diff
@@ -176,6 +176,11 @@ def delete(self, entity_name):
     def read_subscriptions(self):
         """Return subscriptions table"""
         view = self.navigate_to(self, 'All')
+        wait_for(
```
Comment (omkarkhatavkar) [inline airgun/entities/subscription.py:179]: we are trying to read the table 2 times here. which is not a good way. Better way would be just add this line 

`view.wait_displayed()`

**Review by omkarkhatavkar — CHANGES_REQUESTED** [review]
minor change

File: airgun/entities/subscription.py (hunk 1 above)
Comment (sambible) [reply airgun/entities/subscription.py:179]: ah yeah, this is definitely the better way to do this. fuzzy late night brain :) 

If that ends up not being specific enough, you can always use wait_displayed on a specific element, as well.

I.e. `view.table.wait_displayed`

Comment (vijaysawant) [reply airgun/entities/subscription.py:179]: okay

**Review by omkarkhatavkar — APPROVED** [review]

**Review by vsedmik — APPROVED** [review]

**Pull request merged**